src/test/java/spaceinvaders/group_22/PlayerTest.java

```diff
@@ -19,7 +19,8 @@
 	@SuppressWarnings("checkstyle:magicnumber")   
 	public final void testGetSetSpaceShip() {
 		SpaceShip ship = new SpaceShip(10.0, 10.0, "testimage.png");
-		Player player = new Player(new Game(200 , 200));
+		Game game = new Game(200, 200);
+		Player player = new Player(game, game.getCanvasWidth() / 2);
```

Comment:
A better way to do this is change getCanvasWitdth() to 100. Or create an int canvasWidth and also use it to create the game object.